app/networkapi/features/__init__.py

```diff
@@ -0,0 +1 @@
+default_app_config = 'networkapi.features.apps.FeaturesConfig'
```

Comment:
This is the future! It also forces the ready method defined on the AppConfig class to execute when the module is loaded.